sandbox/api/utils.py

```diff
@@ -181,3 +179,82 @@ def generate_response_from_example(example_path: str, status_code: int) -> Respo
     # Value of response is always in the first key, then within value
     content = content[list(content.keys())[0]]["value"]
     return Response(dumps(content), status=status_code, mimetype=FHIR_MIMETYPE)
+
+
+def check_for_consent_include_params(
+    _include: str,
+    logger: logging.Logger,
```

Comment:
Logger shouldn't be passed to the function instead at the top of the file set using `logger = getLogger(__name__)`
same as in app.py